src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -147,6 +156,38 @@ private void processRequests(HttpServerRequest httpServerRequest) {
 
                 break;
 
+            //create a sink endpoint and initialize consumer
+            case CREATE:
+                SinkBridgeEndpoint<?,?> sink = new HttpSinkBridgeEndpoint<>(this.vertx, this.bridgeConfigProperties);
+                sink.closeHandler(s -> {
+                    this.httpSinkEndpoints.remove(sink);
+                });
+
+                sink.open();
+
+                sink.handle(new HttpEndpoint(httpServerRequest), consumerId -> {
+                    this.httpSinkEndpoints.put(consumerId, sink);
+                });
+
+                break;
+
+            case SUBSCRIBE:
+
+                String subscribeInstanceId = PathParamsExtractor.getConsumerSubscriptionParams(httpServerRequest).get("instance-id");
+
+                final SinkBridgeEndpoint subscribeSink = this.httpSinkEndpoints.get(subscribeInstanceId);
+
+                subscribeSink.handle(new HttpEndpoint(httpServerRequest));
+                break;
+
+            case CONSUME:
+
+                String consumeInstanceId = PathParamsExtractor.getConsumerConsumptionParams(httpServerRequest).get("instance-id");
+
+                final SinkBridgeEndpoint consumeSink = this.httpSinkEndpoints.get(consumeInstanceId);
+
+                consumeSink.handle(new HttpEndpoint(httpServerRequest));
+
```

Comment:
there is a `break` missing. Anyway, the `CONSUME` and `SUBSCRIBE` cases are quite the same can we collapse them in only one case? i.e. 

```
case SUBSCRIBE:
case CONSUME:
      String instanceId = ....
      ...
      ...
      break;
```